### ISSUE TITLE
Train with taming's VQGAN

### ISSUE THREAD

**Issue by WocMaker** [lucidrains/DALLE-pytorch#328]
Hi,
I was training on my own data set with taming's VQ-GAN. 
There's some error with the dimension with the vae model
![image](https://user-images.githubusercontent.com/85055246/124013852-68a8e680-da15-11eb-8dea-43bc81519a9f.png)
![image](https://user-images.githubusercontent.com/85055246/124013913-79595c80-da15-11eb-9cab-6c3eaa4ea9bd.png)
Can anyone help me with it? How can I change this part to let it work?

**Comment by lucidrains** [comment]
@rom1504 could this be related to your recent PR?

**Comment by WocMaker** [comment]
> @rom1504 could this be related to your recent PR?
Hi lucidrain,
I had checked rom's dalle server respo before.
I still have the PR.
Do I need to modifiy more, besides adding the args --taming to use VAGAN as vae?
![image](https://user-images.githubusercontent.com/85055246/124087527-aa24aa80-da84-11eb-9a9d-1a60f094c00a.png)

**Comment by WocMaker** [comment]
Since  when I use open AI's vae with my own dataset, the loss is not decreasing.
![image](https://user-images.githubusercontent.com/85055246/124087800-f53ebd80-da84-11eb-9bee-06d786787de3.png)

**Comment by rom1504** [comment]
@lucidrains maybe. I will check today (and fix if there's indeed a regression)

@WocMaker no taming option is indeed all that is needed.

The results with openAI vae are indeed as expected (and as I measured in my own experiments). It's about 4 to 8 times slower to converge when using openAI vae (mostly due to the 1024 seq length).

**Comment by WocMaker** [comment]
@rom1504 
Thanks, Romain. 
Maybe there's some problem with my dataset.
It's werid that my dataset can run with  openAI vae, but it result in dim error for VQ-GAN. May I just ask for double check that, the dataset format needed is the same with these two vae setting up? orz

**Comment by rom1504** [comment]
hi @WocMaker , I don't think there is a problem with your dataset.
There was a recently introduced bug in Dalle-pytorch code, that I fixed in https://github.com/lucidrains/DALLE-pytorch/pull/329
You can directly use this change or wait for it to be merged

**Comment by lucidrains** [comment]
@rom1504 thank you! merged 🙏

**Comment by WocMaker** [comment]
@rom1504 @lucidrains 
Thank you so much!!!

**Comment by Mehrnia7395** [comment]
I have the same problem, I'm using the vae for training on the celeb-A dataset, but the loss is not decreasing.
